contrib/completion/bash/docker

```diff
@@ -1759,7 +1759,7 @@ _docker_daemon() {
 			_filedir
 			return
 			;;
-		--exec-root|--graph|-g)
+		--exec-root|--data-root)
```

Comment:
That sounds reasonable. There's also not a shorthand for the `--exec-root` option. Let me know when the maintainers have a consensus on what to do here and I'll update if needed.